src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -70,62 +87,65 @@
   }
 }
 
-.#{$code-editor}__controls {
+.#{$code-editor}__header-content {
   display: flex;
-
-  .#{$button}.pf-m-control {
-    --#{$button}--m-control--Color: var(--#{$code-editor}__controls--c-button--m-control--Color);
-
-    &:hover {
-      --#{$code-editor}__controls--c-button--m-control--Color: var(--#{$code-editor}__controls--c-button--m-control--hover--Color);
-    }
-
-    &:focus {
-      --#{$code-editor}__controls--c-button--m-control--Color: var(--#{$code-editor}__controls--c-button--m-control--focus--Color);
-    }
-
-    &:disabled {
-      &::after {
-        border-block-end-color: var(--#{$code-editor}__controls--c-button--m-control--disabled--after--BorderBottomColor);
-      }
-    }
+  flex: 1;
+  align-items: flex-end;
+  padding-block-start: var(--#{$code-editor}__header-content--PaddingBlockStart);
+  padding-block-end: var(--#{$code-editor}__header-content--PaddingBlockEnd);
+  padding-inline-start: var(--#{$code-editor}__header-content--PaddingInlineStart);
+  padding-inline-end: var(--#{$code-editor}__header-content--PaddingInlineEnd);
+  background-color: var(--#{$code-editor}__header-content--BackgroundColor);
+  border-radius: var(--#{$code-editor}__tab--BorderRadius) 0 0 0;
+
+  &.pf-m-plain {
+    --#{$code-editor}__header-content--BackgroundColor: var( --#{$code-editor}__header-content--m-plain--BackgroundColor);
```

Comment:
I moved the plain modifier up to `__header`. I think the background color needs to remain on the `header-content` though, because moving it up to the header was creating a grey background when it shouldn't (like when there's no header only the language tab).

Reply:
@kmcfaul thanks, perfect!